src/test/scala/com/cognite/spark/datasource/AssetsRelationTest.scala

```diff
@@ -94,10 +94,11 @@ class AssetsRelationTest extends FlatSpec with Matchers with SparkTest {
         |'asset description' as description,
         |null as metadata,
         |'$assetsTestSource' as source,
-        |null as id,
-        |null as createdTime,
-        |null as lastUpdatedTime
+        |10 as id,
+        |0 as createdTime,
```

Comment:
Is "null" valid here, or do we need the explicit 0 now?

Reply:
Needs to be explicit, `name`, `id`, `createTime` and `lastUpdatedTime` are no longer `Option`s since I'm casting `Row`s to `sdk.scala.Asset`s.

Reply:
Hmm, I wonder if we should manage that in the Spark layer. Maybe we should get user feedback (as well as test it out ourselves and see how we feel about it) first though.